.github/ISSUE_TEMPLATE/submit_question.md

```diff
@@ -0,0 +1,20 @@
+---
+
+name: Submit Question
+about: Ask a general question about pandas
+title: "QST:"
+labels: ""
+
+---
+
+#### Question about pandas
+
+**Note**: It's highly recommended that you use [StackOverflow](https://www.stackoverflow.com) for any usage related questions, e.g. "How do I do [...] with pandas?" You are much more likely to receive a quick response to your question on StackOverflow than the GitHub issue tracker. You may also want to search the [pandas tag](https://stackoverflow.com/questions/tagged/pandas) on StackOverflow to see if a similar question has already been asked and answered.
```

Comment:
@mroeschke I *think* this is very similar to a proposal you made [here](https://github.com/pandas-dev/pandas/pull/33187#discussion_r401165273), would you like me to change that to a checkbox, as well?

Reply:
Sure great point. I think the 2 checkboxes could be something like

```
- [ ] I have searched the [pandas tag] on StackOverflow for similar questions
- [ ] I have asked my usage related question on [StackOverflow]
```